### PR TITLE
Fix `warning: method redefined; discarding old female`

### PR DESCRIPTION
**Pull request by kamipo**
```
$ ARCONN=mysql2 be ruby -w -Itest test/cases/scoping/default_scoping_test.rb
Using mysql2
/Users/kamipo/src/github.com/rails/rails/activerecord/lib/active_record/scoping/named.rb:158: warning: method redefined; discarding old female
/Users/kamipo/src/github.com/rails/rails/activerecord/lib/active_record/scoping/named.rb:158: warning: previous definition of female was here
/Users/kamipo/src/github.com/rails/rails/activerecord/lib/active_record/scoping/named.rb:158: warning: method redefined; discarding old male
/Users/kamipo/src/github.com/rails/rails/activerecord/lib/active_record/scoping/named.rb:158: warning: previous definition of male was here
```

### CODE REVIEW

File: activerecord/test/models/cat.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
 class Cat < ActiveRecord::Base
   self.abstract_class = true
 
-  enum gender: [:female, :male]
-
-  scope :female, -> { where(gender: genders[:female]) }
-  scope :male, -> { where(gender: genders[:male]) }
+  scope :female, -> { where(gender: :female) }
+  scope :male, -> { where(gender: :male) }
```
Comment (jeremy) [inline activerecord/test/models/cat.rb]: Can we simply remove these scopes? The `enum` declaration is creating them already.

Comment (kamipo) [inline activerecord/test/models/cat.rb]: make sense!

File: activerecord/test/models/cat.rb [hunk 2]
```diff
@@ -3,9 +3,6 @@ class Cat < ActiveRecord::Base
 
   enum gender: [:female, :male]
 
-  scope :female, -> { where(gender: genders[:female]) }
-  scope :male, -> { where(gender: genders[:male]) }
```
Comment (kamipo) [inline activerecord/test/models/cat.rb]: Simply removed these scopes.

**Comment by pixeltrix** [comment]
@kamipo thanks :+1:

**Pull request merged**